testcases/bestest_hydronic_heat_pump/models/BESTESTHydronicHeatPump/TestCase.mo

```diff
@@ -879,6 +889,18 @@ https://www.carbonfootprint.com/docs/2019_06_emissions_factors_sources_for_2019_
 <ul>
 <li>
 October 18, 2024, by Ettore Zanetti:<br/>
+Update <code>min</code> and <code>max</code> for ovewrite blocks.
```

Comment:
I don't see that this was done for bestest_hydronic_heat_pump.

Reply:
You are right. I updated the release notes removing this.